### ISSUE TITLE
Add 'Name' to JSON schema

### ISSUE THREAD

**Issue by devblackops** [rnelson0/vCenter-roles#9]
I would suggest adding the Name field to the JSON schema. If the Import-VIRole function supported the pipeline, you could import multiple roles like below. You could also specify the Name explicitly when calling the function to override the name in the JSON file.

``` powershell
dir '*.json' | Import-VIRole -vCenter vcenter01.mydomain.com
```

**Comment by rnelson0** [comment]
Good idea! I will make sure this is addressed as the JSON scheme is updated.
